ydb/core/tx/schemeshard/schemeshard_utils.cpp

```diff
@@ -282,15 +361,14 @@ NKikimrSchemeOp::TTableDescription CalcImplTableDesc(
         implKeyToImplColumn[implTableColumns.Keys[keyId]] = keyId;
```

Comment:
alreadyUsed is 1 for postingImplTable and 0 for other

```suggestion
        implKeyToImplColumn[implTableColumns.Keys[keyId]] = keyId + alreadyUsed;
```

Reply:
The key migration algorithm is rewritten.